hera_mc/cm_active.py

```diff
@@ -205,23 +205,43 @@ def load_geo(self, at_date=None):
             key = cm_utils.make_part_key(ageo.station_name, None)
             self.geo[key] = ageo
 
-    def revs(self, hpn):
+    def revs(self, hpn, exact_match=False):
         """
-        Returns a list of active revisions for a given hpn.
+        Returns a list of active revisions for the provided hpn list.  The returned
+        list is the concatentated set of revisions found for the provided list.
```

Comment:
"And can include more than one entry for each input entry if `exact_match` is not set and multiple parts start with the provided input entry". Right?

Reply:
No - it can provide more than one entry even if there is only one hpn supplied, since there may be more than one active revision.

Reply:
what if the hpn supplied is incomplete (just the first few characters) and can be matched to multiple parts?

Reply:
It provides the concatenation of all.  This is the exact purpose of it.  If I want to know what revisions of a particular part are in the system I query on hpn='PAM' and it tells me all of the relevant revisions.

Reply:
That makes perfect sense, but that is not clear to me at least from the current docstring. I read the docstring and think you get one active revision per item in the input list.

Reply:
It has to check all of them to check for all active revisions.  I can also check on a single part etc.